.eslintrc.js

```diff
@@ -12,6 +12,8 @@ module.exports = {
         "import/resolver": {
             alias: {
                 map: [
+                    ["@task", "./task"],
```

Comment:
Very  good, I just had an issue with the npm package that is generated, the links are broken.  But that is another issue